dev/source/docs/using-sitl-for-ardupilot-testing.rst

```diff
@@ -375,10 +375,14 @@ Testing Compass Calibration
 A quick way to test compass calibration in SITL is with the
 "calibration" vehicle model. To use this with plane do this:
 
```

Comment:
Just FYI, you can put the :: at the end of the last content line too, or use `.. code-block::` directive (not useful here but can be if you have a programming language block)